### DISCUSSION TITLE
Slider change icon color handling

### DISCUSSION THREAD

**Discussion by bartekev** [Clooos/Bubble-Card#1020] [Questions about config, custom styles and templates]
Pretty sure there's some easy way to do it, but can't seem to find it.
I would like the icon of slider to have the same behavior as the standard tile card: 
* turned off: icon grey on black background
* turned on: icon yellow-orange-ish with soft background
Can this be easily configured?

**Comment by MrBearPresident** [comment]
For something like this:
![2024-12-19_09-06-33](https://github.com/user-attachments/assets/98d45859-f50b-4365-a8b1-caa4fea7ebc9)


Add this to your custom style:

```
:host{
  --WantedColor: 68, 115, 158;
}

.bubble-icon-container:before{
  content: "";
  background: ${ state > 0 ? 'rgba(var(--WantedColor),0.4)' :''};
  position: absolute !important;
  border-radius: 50% !important;
  height: 42px;
  width: 42px;
  }

.bubble-icon-container .bubble-icon{  
  color: ${ state > 0 ? 'rgba(var(--WantedColor),1)' :''} !important;
  }
```

---
<sup>If this solves your problem, mark this as the solution and close the discussion.</sup>
